Users/Glob_Patterns.rst

```diff
@@ -123,14 +121,10 @@ brackets. Parentheses that have no match are ignored as well as
     False
```

Comment:
It's not a kind of error, I think. and as far as its output is concerned it's explained in the docs.